README.md

```diff
@@ -6,13 +6,13 @@
 
 Flowise is an open source low-code tool for developers to build customized LLM orchestration flow & AI agents.
 
-Watch a quickstart demo (2mins)
+### [Watch a quickstart demo (2 mins)](https://youtu.be/d7vfUodP0c4)
```

Comment:
thanks for the spelling & grammar fix!

though I'd still prefer the original readme page:

![image](https://github.com/FlowiseAI/FlowiseDocs/assets/26460777/2d1dc650-c821-428c-9122-7a4bc8828f17)

![image](https://github.com/FlowiseAI/FlowiseDocs/assets/26460777/8c478fed-4dc2-4de5-9a1b-7741fc26223f)

just to be consistent with other pages